frontend/src/container/PersonalRecentActivities.jsx

```diff
@@ -33,6 +33,7 @@ export class PersonalRecentActivities extends React.Component {
     props.registerCustomEventHandlerList([
       { name: CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, handler: this.handleAllAppChangeLanguage }
     ])
+    this.isLoadMoreIsProgress = false
```

Comment:
`isLoadMoreIsProgress`, looks like a typo to me, but since you put it all way long I'm not sure... Shouldn't be `isLoadMoreInProgress`?